packages/block-editor/src/components/block-preview/auto.js

```diff
@@ -46,7 +45,7 @@ function ScaledBlockPreview( {
 		if ( styles ) {
 			return [
 				...styles,
-				...__experimentalStyles,
+				...additionalStyles,
```

Comment:
That's not from this PR, but is there a case that we don't have `styles` but we do have `additionalStyles`?

Reply:
Yes in theory there could be no `styles` attribute in the block editor settings which would mean `styles` is `undefined` here.